packages/docs/examples/va-progress-bar/Coloring.vue

```diff
@@ -0,0 +1,21 @@
+<template>
+  <div class="flex lg6 xs12">
+    <div class="my-2 text--bold muted">Danger color:</div>
+    <va-progress-bar
+      value="35"
+      color="danger"
+    />
+    <div class="my-2 text--bold muted">Custom color:</div>
+    <va-progress-bar
+      value="35"
+      color="purple"
+    />
+    <div class="mb-4" />
+  </div>
+</template>
+
+<style scoped>
+.muted {
```

Comment:
We really do not want to introduce any unneded styling to examples.
Maybe move to global class?